pegasus/sites.v3/code.org/views/learn_carousels.haml

```diff
@@ -20,12 +22,18 @@
     %button{:style=>"font-size:20px;"}
       = I18n.t(:carousel_heading_domestic)
 
-#all-tutorials{:style=>'visibility:hidden; overflow:hidden;'}
-  - categories.each do |category|
-    - id = "collection-" + category[:tag]
-    = view :learn_carousel, :heading=>category[:name], :tag=>category[:tag], :tutorials=>tutorials, :id=>id
+-if variation == 'learn'
+  #all-tutorials{:style=>'visibility:hidden; overflow:hidden;'}
+    - learn_categories.each do |category|
+      - id = "collection-" + category[:tag]
+      = view :learn_carousel, :heading=>category[:name], :tag=>category[:tag], :tutorials=>tutorials, :id=>id
 
 -if variation == 'beyond'
+  #all-tutorials{:style=>'visibility:hidden; overflow:hidden;'}
+    - beyond_categories.each do |category|
```

Comment:
What do you mean set by a variable? I can't do `"#{variation}_categories".each` so I would have to have at least 3 lines to do if variation X -> loop with this or that, which seemed to me a lot messier to put that under #all-tutorials instead of just showing that they do basically the same thing with different arrays.